### PR TITLE
V16 scrub

### PR DESCRIPTION
**Pull request by zemccartney**
Bringing things as up-to-date as possible while staying Hapi 16 compatible

- Pointed API docs at docs on website, not github, to ensure people get to the right version of the docs
- Spacing changes are due to new complaints from the linter
- Included package-lock, wasn't sure what our stance is there; Remove?

🙏

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "labbable",
-  "version": "2.1.1",
+  "version": "2.1.2",
```
Comment (devinivy) [inline package.json]: Can you unbump this?

File: README.md [hunk 2]
```diff
@@ -4,10 +4,14 @@ No-fuss hapi server testing
...
 
 ##### Why initialize the server for tests?
-Plugin dependencies are only enforced at the time of [server initialization](https://github.com/hapijs/hapi/blob/master/API.md#serverinitializecallback).  This means code that relies on a plugin being present (typically by the `after` callback of [`server.dependency(deps, after)`](https://github.com/hapijs/hapi/blob/master/API.md#serverdependencydependencies-after)) will only run during initialization.  And if there are any dependencies missing, those errors will surface only during initialization.  Your server's caches will also be started and `onPreStart` server extensions will run.
+Plugin dependencies are only enforced at the time of [server initialization](https://hapijs.com/api#serverinitializecallback).  This means code that relies on a plugin being present (typically by the `after` callback of [`server.dependency(deps, after)`](https://hapijs.com/api#serverdependencydependencies-after) will only run during initialization.  And if there are any dependencies missing, those errors will surface only during initialization.  Your server's caches will also be started and `onPreStart` server extensions will run.
```
Comment (devinivy) [inline README.md]: Elsewhere we've started referencing the v16 branch for links rather than the hapi site– can we keep doing that?

**Comment by devinivy** [comment]
Could you also please update .travis.yml?

File: package.json (hunk 1 above)
Comment (zemccartney) [reply package.json]: yessir 🙏

File: README.md (hunk 2 above)
Comment (zemccartney) [reply README.md]: Dah! yes, thanks for catching that

**Pull request merged**